lib/Wikia/src/Service/User/Preferences/PreferenceModule.php

```diff
@@ -14,12 +15,17 @@
 
 class PreferenceModule implements Module {
 	const PREFERENCE_CACHE_VERSION = 3;
+	const PREFERENCE_CACHE = false;
 
 	public function configure( InjectorBuilder $builder ) {
 		$builder
 			->bind( PreferenceService::class )->toClass( PreferenceServiceImpl::class )
 			->bind( PreferencePersistence::class )->toClass( PreferencePersistenceSwaggerService::class )
 			->bind( PreferenceServiceImpl::CACHE_PROVIDER )->to( function() {
+				if ( !self::PREFERENCE_CACHE ) {
```

Comment:
There should be no logic inside binding modules.
This is anti pattern and should be avoided if possible.